Vagrantfile

```diff
@@ -82,65 +83,55 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # EOL January 2018
   # Only kept around for those few dev's who have already got this image and continue to use it.
   config.vm.define "zesty32", autostart: false do |zesty32|
-    config.vm.box = "ubuntu/zesty32"
-    config.vm.provision :shell, path: "Tools/vagrant/initvagrant.sh"
-    config.vm.provider "virtualbox" do |vb|
+    zesty32.vm.box = "ubuntu/zesty32"
+    zesty32.vm.provision :shell, path: "Tools/vagrant/initvagrant.sh"
+    zesty32.vm.provider "virtualbox" do |vb|
       vb.name = "ArduPilot (zesty32)"
     end
   end
 
   # 17.10, EOL July 2018
   # Only kept around for those few dev's who have already got this image and continue to use it; not available for download
   config.vm.define "artful32", autostart: false do |artful32|
-    config.vm.box = "ubuntu/artful32"
-    config.vm.provision :shell, path: "Tools/vagrant/initvagrant.sh"
-    config.vm.provider "virtualbox" do |vb|
+    artful32.vm.box = "ubuntu/artful32"
+    artful32.vm.provision :shell, path: "Tools/vagrant/initvagrant.sh"
+    artful32.vm.provider "virtualbox" do |vb|
       vb.name = "ArduPilot (artful32)"
     end
   end
 
-  # 18.04 LTS
```

Comment:
We prefer to leave these in as people may still have boxes around they use.

Reply:
Fixed